services/openvpn/service/manager.go

```diff
@@ -103,12 +102,16 @@ func (m *Manager) Serve(providerID identity.Identity) (err error) {
 		IP:   net.ParseIP(m.serviceOptions.Subnet),
 		Mask: net.IPMask(net.ParseIP(m.serviceOptions.Netmask).To4()),
 	}
-	err = m.natService.Add(nat.RuleForwarding{
-		SourceSubnet: m.vpnNetwork.String(),
-		TargetIP:     m.outboundIP,
-	})
-	if err != nil {
-		return errors.Wrap(err, "failed to add NAT forwarding rule")
+
+	var dnsOK bool
+	var dnsIP net.IP
+	var dnsPort = 11153
```

Comment:
Would it be possible to allow user to set dnsPort = 53? For example If I run in datacenter I can run provider node with dns on 53 port.